### DISCUSSION TITLE
Functor 用途

### DISCUSSION THREAD

**Discussion by dannypsnl** [FizzyElt/functional-programming#70] [Q&A] [answered]
https://fizzyelt.github.io/functional-programming/fp-000G.xml

我對這篇實在不理解到底要說啥

是不是跟新的程式 functor vs 範疇 functor 有關？

**Comment by FizzyElt** [comment]
主要是想闡述一個 Functor 能對我們實際開發起到什麼作用
不過當時沒想這麼多，寫的怪怪的

**Comment by FizzyElt** [comment, marked as answer]
以現在來說要擺在 "程式 functor vs 範疇 functor" 這段後面，至於內容可能要重新想一下

**Comment by dannypsnl** [comment]
另外我可以開一篇講 F-algebra 了，因為有 CCC 那個 PR 了，合併之後我就可以整理一下

**Comment by dannypsnl** [comment]
<img width="538" alt="image" src="https://github.com/FizzyElt/functional-programming/assets/22004511/8d210ec3-4580-4be9-a125-29b042a67501">
